graph-db/src/utils/cmr/__test__/harvestCmrCollections.test.js

```diff
@@ -0,0 +1,39 @@
+const nock = require('nock')
+const { harvestCmrCollections } = require('../harvestCmrCollections')
+let { fetchPageFromCMR } = require('../fetchPageFromCmr')
+const { expect } = require('@jest/globals')
```

Comment:
You shouldn't need to import this when jest is installed locally as part of your package.json.